### ISSUE TITLE
WARNING: Unknown module

### ISSUE THREAD

**Issue by gregw** [eclipse/jetty.project#10421]
**Jetty version(s)**
12

**Description**

In a clean successful build (e.g. [1482](https://jenkins.webtide.net/blue/rest/organizations/jenkins/pipelines/jetty.project/branches/jetty-12.0.x/runs/1482/nodes/7/steps/33/log/?start=0) there are at least 15 warnings like:
```
WARNING: Unknown module: org.eclipse.jetty.http2.server specified to --add-reads
```

**Comment by olamy** [comment]
I can see only one warning
```
wget -O out.txt "https://jenkins.webtide.net/blue/rest/organizations/jenkins/pipelines/jetty.project/branches/jetty-12.0.x/runs/1482/nodes/7/steps/33/log/?start=0" 


olamy@pop-os:~/dev/sources/jetty/jetty.project$ grep -c "org.eclipse.jetty.http2.server specified to --add-reads" out.txt 
1
```
:)

**Comment by gregw** [comment]
fixed by #10424

**Comment by olamy** [comment]
fixed with https://github.com/eclipse/jetty.project/pull/10424
